src/store/Network.ts

```diff
@@ -66,30 +67,37 @@ const staticPeerNodes: NodeInfo[] = [
     {
         nodePublicKey: 'BE60BE426872B3CB46FE2C9BAA521731EA52C0D57E004FC7C84293887AC3BAD0',
         host: 'beacon-01.ap-northeast-1.0.10.0.x.symboldev.network',
+        roles: [RoleType.PeerNode],
```

Comment:
will this get filtered out anyway?

I guess we should keep these static nodes until testnet refresh.

Reply:
No, it won't. We are keeping the following role types: [PeerNode, VotingNode] and filtering out the rest. So static nodes will be there.
![Screenshot 2020-11-20 at 17 45 28](https://user-images.githubusercontent.com/6256269/99832625-bcd31a80-2b58-11eb-8c14-f5ca4b8830f1.png)